packages/jest-snapshot/src/index.ts

```diff
@@ -153,11 +153,10 @@ export const cleanup = (
   };
 };
 
-export const toMatchSnapshot: MatcherFunctionWithContext<Context> = function (
-  received: unknown,
-  propertiesOrHint?: object | string,
-  hint?: string,
-) {
+export const toMatchSnapshot: MatcherFunctionWithContext<
+  Context,
+  [propertiesOrHint?: object | string, hint?: string]
+> = function (received, propertiesOrHint, hint) {
```

Comment:
Here and bellow – types in function signature is the only change. The rest is white space / formatting.